zel/src/org/zkoss/zel/impl/util/ReflectionUtil.java

```diff
@@ -43,6 +48,10 @@
             byte.class, char.class, double.class, float.class, int.class,
             long.class, short.class, Void.TYPE };
 
+	//for reflection cache
+	private static final Method[] EMPTY_METHOD_ARRAY = new Method[0];
+	private static final Map<Class<?>, Map<String, Method[]>> SETTERS_METHODS_CACHE = new ConcurrentHashMap<Class<?>, Map<String, Method[]>>();

```

Comment:
@DevChu do you need a different initial capacity other than the default 16 for the SETTERS_METHODS_CACHE??